components/x-live-blog-post/src/svgIcons.jsx

```diff
@@ -6,8 +6,8 @@ Once this is resolved, these components should be replaced with o-share https://
  */
 export function TwitterSVG() {
```

Comment:
**Non-blocking suggestion:** Should we update this function name as well? Side-note, `X` is such an awkward name to use in functions 🙈

Reply:
Ah, just realise this is probably used externally, so probably a lot of effort to rename it!